src/sedcli_main.c

```diff
@@ -113,6 +115,14 @@ static cli_option blocksid_opts[] = {
 	{0}
 };
 
+static cli_option add_user_lr_opts[] = {
+	{'d', "device", "Device node e.g. /dev/nvme0n1", 1, "DEVICE", CLI_OPTION_REQUIRED},
+	{'u', "user", "String specifying the user to be added to the LR {userN, N=[1..9]}", 1, "FMT", CLI_OPTION_REQUIRED},
+	{'t', "accesstype", "String specifying access type to the data on drive. Allowed values: RO/RW/LK", 1, "FMT", CLI_OPTION_REQUIRED},
+	{'l', "locking-range", "Locking Range {1..9}", 1, "NUM", CLI_OPTION_OPTIONAL_ARG},
```

Comment:
looks like "locking-range" is optional and you allow values from 1 to 9 as valid input. by default locking-range would be set to 0 which seems to be invalid.